### PR TITLE
Jenkins fixes

### PR DESCRIPTION
**Pull request by liskin**
Two fixes for Jenkins, see the discussion at https://github.com/ocharles/tasty-ant-xml/commit/a01df06b59122c3086fc9f42854129b1d4a8c31c#commitcomment-21211275

### CODE REVIEW

File: Test/Tasty/Runners/AntXML.hs [hunk 1]
```diff
@@ -130,11 +131,14 @@ antXMLRunner = Tasty.TestReporter optionDescription runner
           Const summary <$ State.modify (+ 1)
 
         runGroup groupName children = Tasty.Traversal $ Functor.Compose $ do
-          Const soFar <- Reader.withReaderT (++ [groupName]) $ Functor.getCompose $ Tasty.getTraversal children
+          Const soFar <- Reader.local (groupName :) $ Functor.getCompose $ Tasty.getTraversal children
```
Comment (haishengwu-okta) [inline Test/Tasty/Runners/AntXML.hs]: nit:
pre-append groupName then `reverse` seems a bit waist rather than append to end.

Comment (liskin) [inline Test/Tasty/Runners/AntXML.hs]: Appending to a list is generally frowned upon for its computational complexity, but in this case it probably wouldn't be noticeable.

**Comment by haishengwu-okta** [comment]
@ocharles just in case he missed..

**Comment by ocharles** [comment]
Thanks for the reminder!

**Comment by ocharles** [comment]
Released in `tasty-ant-xml-1.1.0`.

**Pull request merged**